go/tasks/plugins/k8s/kfoperators/pytorch/config.go

```diff
@@ -0,0 +1,32 @@
+package pytorch
+
+import (
+	"time"
+
+	pluginsConfig "github.com/flyteorg/flyteplugins/go/tasks/config"
+	"github.com/flyteorg/flytestdlib/config"
+)
+
+//go:generate pflags Config --default-var=defaultConfig
+
+var (
+	defaultConfig = Config{
+		Timeout: config.Duration{Duration: 1 * time.Minute},
+	}
```

Comment:
qq is this functionality the same for all kfoperator plugins? should we add a config for timeout on tensorflow and mpi as well?